gen/suites/msg_application/invalid_msgs.go

```diff
@@ -36,3 +37,15 @@ func failInvalidReceiverMethod(v *MessageVectorBuilder) {
 
 	v.Assert.EveryMessageResultSatisfies(ExitCode(exitcode.SysErrInvalidMethod))
 }
+
+func failInvalidSenderNonAccountActor(v *MessageVectorBuilder) {
+	v.Messages.SetDefaults(GasLimit(1_000_000_000), GasPremium(1), GasFeeCap(200))
+
+	alice := v.Actors.Account(address.SECP256K1, balance1T)
+	v.CommitPreconditions()
+
+	v.Messages.Sugar().Transfer(builtin.SystemActorAddr, alice.ID, Nonce(0), Value(transferAmnt))
```

Comment:
while sending from system is disallowed, might also be worth validating that sending from a storageMiner type address is disallowed, since that's one where a party could plausibly have a key to properly sign the thing?